x/uniq.js

```diff
@@ -1,6 +1,4 @@
 const isArray = require('../_internal/isArray')
-const reduce = require('../reduce')
-
 /**
```

Comment:
lint i think

Reply:
force pushed, hopefully this will be ok now.